src/language/experimentalOnlineParser/rules.json

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
It looks like Deno doesn't support importing JSON:
https://github.com/denoland/deno/issues/5633

Can you please convert it to `rules.js` and also merge it with `grammarTypes.js`?
On the plus side, we would be able to type-check rules content with TS.
Also please name the resulting file `grammar.js`.

Reply:
If it triggers a coverage check please add necessary files to list of excluded files inside `.nycrc`.

Reply:
I'll update.